### ISSUE TITLE
question about training step

### ISSUE THREAD

**Issue by crazy-stycxj** [ken2576/vision-nerf#20]
HI! Great Work！
I find the global_step=500000 ，when i trained last time ，the code died，then start_step=100000，not 0，if i train the network again，i need train if global_step > model.start_step + args.n_iters + 1:     600000steps，why not  400000steps，Will this overfit？Isn't 500000step a reasonable training interval?
please answer me ，thanks

**Comment by ken2576** [comment]
Hi

So I think that part of the code makes it additive for the number of steps to train.
For example, if you have a checkpoint at 10k and set the `n_iters` to be 50k, then the total training steps will be 60k instead of 50k.
As for overfitting, it should be fine because at that point the learning rate will be very small. I didn't test more than 500k steps so you might need to check the eval PSNR and see if it goes back down (overfitting).

**Comment by crazy-stycxj** [comment]
OK，thanks
